Kiss.Bff.EndToEndTest/NieuwsEnWerkInstructies/Helpers/Locators.cs

```diff
@@ -10,8 +10,17 @@ internal static class Locators
         public static ILocator GetSkillsFieldset(this IPage page) => page.GetByRole(AriaRole.Group, new() { Name = "Filter op categorie" });
         public static ILocator GetWerkberichtTypeSelector(this IPage page) => page.Locator($"#werkbericht-type-input");
         public static ILocator GetNieuwsAndWerkinstructiesSearch(this IPage page) => page.Locator($"#search-input"); 
+       
         public static ILocator GetSearchResult(this IPage page) => page.Locator("section").Filter(new() { Has = page.GetByRole(AriaRole.Heading, new() { Name = "Zoekresultaten" }) });
         public static ILocator GetSearchResultFilteredByType(this IPage page,string type) => page.GetSearchResult().GetByRole(AriaRole.Article).Filter(new() { Has = page.Locator("small", new() { HasText = type }) });
 
-    } 
+        public static ILocator GetBeheerRowByValue(this IPage page, string title) => page.GetByRole(AriaRole.Row)
+                      .Filter(new()
+                      {
+                          Has = page.GetByRole(AriaRole.Cell, new() { Name = title, Exact = true }).First
+                      });
+        public static ILocator GetBeheerTableCell(this IPage page, int col, int row) =>
```

Comment:
If we need to find a table cell by column number and row number, maybe the table is not accessible enough. Fixing these type of issues is not within the scope of this story. I think it would be good to make a note of these kind of findings, for instance in the readme of the test project. What do you think, @mstokericatt?

Reply:
if we put them in the read me we'll never do anything with them. I would say: if it's a small adjustment just do it as part of the test story. if it's a big change make a story for it on the backlog.
How could we make this table more accessible?

Reply:
@felixcicatt shall we leave this as it is for now and setup some guidelines for locators and html accessibility some other time?

Reply:
Agreed. Also, I've looked into it a bit and I think playwright doesn't offer a way to select a cell by it's column header